go/consensus/cometbft/apps/keymanager/churp/ext.go

```diff
@@ -79,6 +88,16 @@ func (*churpExt) EndBlock(*tmapi.Context) error {
 	return nil
 }
 
+// EndBlock implements api.Extension.
 func (ext *churpExt) InitChain(*tmapi.Context, types.RequestInitChain, *genesis.Document) error {
 	return nil
 }
+
+func (*churpExt) enabled(ctx *tmapi.Context) (bool, error) {
+	regState := registryState.NewMutableState(ctx.State())
+	regParams, err := regState.ConsensusParameters(ctx)
```

Comment:
Why is this not in the key manager parameters? I guess because now all of the parameters are only for the secrets part and there are no global parameters for the key manager backend?

Reply:
Yes, currently there are no global parameters and in the registry we already have `DisableKeyManagerRuntimeRegistration`. Again, this could be removed after the upgrade.